backend/FwLite/LcmCrdt/Changes/Entries/AddEntryComponentChange.cs

```diff
@@ -9,22 +7,26 @@ namespace LcmCrdt.Changes.Entries;
 
 public class AddEntryComponentChange : CreateChange<ComplexFormComponent>, ISelfNamedType<AddEntryComponentChange>
 {
+    public double Order { get; }
     public Guid ComplexFormEntryId { get; }
     public Guid ComponentEntryId { get; }
     public Guid? ComponentSenseId { get; }
 
     [JsonConstructor]
     public AddEntryComponentChange(Guid entityId,
+        double order,
```

Comment:
adding this as a required parameter will break any existing db as it won't be able to deserialize into this type. It looks like the default value in the db is 0, maybe we just do that here too? I probably need to figure out how to test this so it doesn't catch us out later.

Reply:
looks like I'm incorrect about this. The order property isn't marked as required, so the serialize will use a default value here if it's not in the json.